mods/creative/inventory.lua

```diff
@@ -172,9 +180,5 @@ creative.register_tab("craftitems", "Items", minetest.registered_craftitems)
 
 local old_homepage_name = sfinv.get_homepage_name
 function sfinv.get_homepage_name(player)
-	if minetest.setting_getbool("creative_mode") then
-		return "creative:all"
-	else
-		return old_homepage_name(player)
-	end
+	return creative_mode and "creative:all" or old_homepage_name(player)
```

Comment:
Here the result is the same because `b` which is `"creative:all"` evaluates to a truth-y value; however, this is not always the case and leads this way to write ternary conditionals misleading. Moreover, this is only a style change which I would prefer you do not include (precisely for the reason that avoiding ternaries when possible, especially if it is a syntax not designed for ternaries).